### PR TITLE
Add manifest.yml to the files list

### PR DESCRIPTION
**Pull request by siakaramalegos**
I couldn't figure out why my package wouldn't work from npm, and the culprit was that the manifest.yml was not being picked up in the package. I replaced the existing line since it did not seem to do anything.

🎉 Thanks for sending this pull request! 🎉

Please make sure the title is clear and descriptive.

If you are fixing a typo or documentation, please skip these instructions.

Otherwise please fill in the sections below.

**Which problem is this pull request solving?**

Example: I'm always frustrated when [...]

**List other issues or pull requests related to this problem**

Example: This fixes #5012

**Describe the solution you've chosen**

Example: I've fixed this by [...]

**Describe alternatives you've considered**

Example: Another solution would be [...]

**Checklist**

Please add a `x` inside each checkbox:

- [ ] I have read the [contribution guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have added tests (we are enforcing 100% test coverage).
- [ ] I have added documentation in the `README.md`, the `docs` directory (if
      any) and the `examples` directory (if any).
- [ ] The status checks are successful (continuous integration). Those can be
      seen below.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "files": [
     "src",
-    "!*~"
+    "manifest.yml"
```
Comment (ehmicky) [inline package.json:8]: `"!*~"` should be kept.

**Review by ehmicky — APPROVED** [review]
Hi @siakaramalegos,
Great call, thanks a lot for this fix!

**Review by ehmicky — CHANGES_REQUESTED** [review]
Only one small change (see comment above).

**Comment by siakaramalegos** [comment]
@ehmicky Thanks for explaining that one! Updated.

**Review by ehmicky — APPROVED** [review]

**Pull request merged**